### PR TITLE
story/EUI-4993 hearing view edit

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4993

### Change description ###
View a Waiting to be listed hearing-Display  

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: api/hearings/models/hearingResponse.model.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ export interface HearingResponseModel {
   receivedDateTime: string;
   responseVersion: number;
   laCaseStatus: string;
+  listingStatus: string;
```
Comment (rayliangatos) [inline api/hearings/models/hearingResponse.model.ts:8]: could these two be enum type according to the API spec.

File: api/hearings/models/hearings.enum.ts [hunk 2]
```diff
@@ -31,6 +40,9 @@ export enum HearingListingStatusEnum {
   VACATED = 'VACATED',
   AWAITING_ACTUALS = 'AWAITING ACTUALS',
   ADJOURNED = 'ADJOURNED',
+  DRAFT = 'DRAFT',
+  PROVISIONAL = 'PROVISIONAL',
+  FIXED = 'FIXED',
```
Comment (rayliangatos) [inline api/hearings/models/hearings.enum.ts:45]: could these three be another kind of enum type?

File: src/hearings/converters/date-request-submitted.answer.converter.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { State } from '../store';
+import { AnswerConverter } from './answer.converter';
+
+export class DateRequestSubmittedAnswerConverter implements AnswerConverter {
+  public transformAnswer(hearingState$: Observable<State>): Observable<string> {
+    return hearingState$.pipe(
+      map(state => {
+        const hearingResponse = state.hearingRequest.hearingRequestMainModel.hearingResponse;
+        const receivedDateTime = moment(hearingResponse.receivedDateTime).format(HearingDateEnum.DisplayMonth);
+        return receivedDateTime;
```
Comment (rayliangatos) [inline src/hearings/converters/date-request-submitted.answer.converter.ts:14]: as chat yesterday this needs to be RequestDetailsModel.requestTimeStamp.

File: src/hearings/hearing.test.data.ts [hunk 4]
```diff
@@ -1,7 +1,8 @@
-import {HearingActualsMainModel} from './models/hearingActualsMainModel';
-import {HearingRequestMainModel} from './models/hearingRequestMain.model';
-import {CategoryType, HearingListingStatusEnum, HearingResult, HMCStatus, PartyType} from './models/hearings.enum';
-import {ServiceHearingValuesModel} from './models/serviceHearingValues.model';
+import { LaCaseStatus } from 'api/hearings/models/hearings.enum';
```
Comment (rayliangatos) [inline src/hearings/hearing.test.data.ts:1]: could this refer to src

**Review by johnbenjamin-hmcts — APPROVED** [review]
Approved

File: src/hearings/models/hearings.enum.ts [hunk 5]
```diff
@@ -31,6 +31,18 @@ export enum HearingListingStatusEnum {
   VACATED = 'VACATED',
   AWAITING_ACTUALS = 'AWAITING ACTUALS',
   ADJOURNED = 'ADJOURNED',
+  DRAFT = 'DRAFT',
+  PROVISIONAL = 'PROVISIONAL',
+  FIXED = 'FIXED'
```
Comment (rayliangatos) [inline src/hearings/models/hearings.enum.ts:36]: These seems different kind of type.

File: src/hearings/converters/date-request-submitted.answer.converter.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import { HearingDateEnum } from '../models/hearings.enum';
+import { State } from '../store';
+import { AnswerConverter } from './answer.converter';
+
+export class DateRequestSubmittedAnswerConverter implements AnswerConverter {
+  public transformAnswer(hearingState$: Observable<State>): Observable<string> {
+    return hearingState$.pipe(
+      map(state => {
+        const hearingResponse = state.hearingRequest.hearingRequestMainModel.requestDetails;
+        const receivedDateTime = moment(hearingResponse.requestTimeStamp).format(HearingDateEnum.DisplayMonth);
```
Comment (rayliangatos) [inline src/hearings/converters/date-request-submitted.answer.converter.ts:13]: this should be const requestDateTime?

**Review by rayliangatos — APPROVED** [review]
Approved with small comment

**Pull request merged**